lib/process_settings/testing/helpers.rb

```diff
@@ -60,12 +60,18 @@ module Helpers
 
         class << self
           def included(including_klass)
-            including_klass.define_method(:teardown) do
```

Comment:
`definie_method(:teardown)` does not allow the user to call `super` and also completely overrides the class's `teardown` method if it already exists.